src/components/policy/policy_external/include/policy/policy_table/types.h

```diff
@@ -682,6 +683,7 @@ struct VehicleDataItem : CompositeType {
    * @return true if type is valid.
    */
   bool ValidateTypes() const;
+  bool ValidateDefault() const;
```

Comment:
Would be nice to have a function brief here and which includes a note stating that this function does not validate enum values